Feature/lora report artifact

m3d opened the pull request:
This is the first step towards reporting artifacts also during their discovery. Surprisingly the change was relatively simple, only the Control Center needs better handling. At the moment it receives the data, print it into console:
```
recording started
continue mission
['TYPE_BACKPACK', 3506, -18369, -752]
['TYPE_BACKPACK', -3902, -19879, -1141]
go home
['TYPE_BACKPACK', 5052, -11916, -414]
stop mission
```
and draw (x,y) of robot (should be different).

zwn commented on subt/control_center_qt.py:
```diff
@@ -197,7 +197,13 @@ def run(self):
+                else:
+                    artf, x, y, z = pose_or_artf
+                    heading = 0
+                    print(pose_or_artf)
```
This is ugly hack. The reported artifact should come into the control center on its own channel.

m3d commented:
I added some commits for transition to message recognition in LoRa driver, but ... it has to be tested on real machine before merge.

m3d commented:
OK fixed, rebased on master and added some fixes for Control Center + replay functionality.
(hmm it seems to me that this was not submitted to github :(

m3d commented:
I am merging it now - new functionality is in replay and can be modified in next PR.
